crates/egui_glow/src/shader/vertex.glsl

```diff
@@ -9,8 +9,14 @@
 #endif
 
 #ifdef GL_ES
+// To avoid weird distortion issues when rendering text etc, we want highp if possible.
+// But apparently some devices don't support it, so we have to check first.
+#if defined(GL_FRAGMENT_PRECISION_HIGH) && GL_FRAGMENT_PRECISION_HIGH == 1
+    precision highp float;
+#else
     precision mediump float;
 #endif
+#endif
```

Comment:
```suggestion
#ifdef GL_ES
    // To avoid weird distortion issues when rendering text etc, we want highp if possible.
    // But apparently some devices don't support it, so we have to check first.
    #if defined(GL_FRAGMENT_PRECISION_HIGH) && GL_FRAGMENT_PRECISION_HIGH == 1
        precision highp float;
    #else
        precision mediump float;
    #endif
#endif
```